packages/technical-features/application/agnostic/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@k8slens/application",
   "private": false,
-  "version": "6.5.0-alpha.3",
+  "version": "6.5.0-alpha.4",
```

Comment:
does everything get automatically bumped? Seems odd that there are a variety of 6.5.0-alpha.x versions(?) Or is it actually reflective of code changes?

Reply:
This bump is due to https://github.com/lensapp/lens/commit/1c48dda707884a81bcb56f647a0274962618921b which was part of https://github.com/lensapp/lens/pull/7437.

So not necessarily as code change, but a dependency update.

And not everything gets automatically updated. Only packages that have changes to their files.